com.adsbynimbus.nimbus/Editor/NimbusSDKDependencies.xml

```diff
@@ -0,0 +1,86 @@
+<dependencies>
+
+  <!-- Android dependencies are specified under the "androidPackages" element.
```

Comment:
Since this file is part of the SDK and not the sample app it's probably best to strip these comments and only use comments for things that deviate from the standards.